tutorials/tutorial11_traffic_lights.ipynb

```diff
@@ -115,9 +127,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Once the TrafficLightParams class is instantiated, traffic lights can be added via the classes' `add` function. One prerequisite of using this function is knowing the node id of any node you intend to manipulate. This information is baked into the experiment's scenario class, as well as the experiment's nod.xml file. For the experiment we are using with 2 rows and 3 columns, there are 6 nodes: \"center0\" to \"center5\".\n",
-    "\n",
-    "In this particular example, each of the 6 traffic light nodes corresponds to the same set of possible phases; in other words, at any time, each node will be at the same phase. You can, however, customize each traffic light node to have different phases."
+    "Once the `TrafficLightParams` class is instantiated, traffic lights can be added via the `add` function. One prerequisite of using this function is knowing the node id of any node you intend to manipulate. This information is baked into the experiment's scenario class, as well as the experiment's `nod.xml` file. For the experiment we are using with 2 rows and 3 columns, there are 6 nodes: \"center0\" to \"center5\"."
```

Comment:
nit: node.xml I think?

Reply:
Also, can you describe the relative ordering of the nodes e.g. center0 is the bottom left, center2 is the bottom right, etc?

Reply:
I think it's actually nod.xml